### PR TITLE
feat: add profile page, sidebar, footer, forgot password

### LINKED ISSUE #12: Create sidebar & footer

### LINKED ISSUE #33: Profile Screen

### LINKED ISSUE #49: Sidebar

### PR DESCRIPTION
**Pull request by KeN095**
closes #33 
closes #49 
closes #12

### CODE REVIEW

**Review by juaoose — APPROVED** [review]
It would be nice if you added the redirects for the forgot password and profile pages.

**Comment by Leeoku** [comment]
sidebar is fine. My only suggestion before merging it is to make it not cover the navbar

**Review by Leeoku — APPROVED** [review]

**Comment by KeN095** [comment]
Will address those issues in the next commit before i push.

File: src/component/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Link, Typography, Grid } from "@material-ui/core";
+import Divider from "@mui/material/Divider";
+import InstagramIcon from "@mui/icons-material/Instagram";
+import TwitterIcon from "@mui/icons-material/Twitter";
+import FacebookIcon from "@mui/icons-material/Facebook";
+
+const copyright = new Date().getFullYear() + " MuxLabs";
```
Comment (juaoose) [inline src/component/Footer.tsx:7]:
```suggestion
const copyright = new Date().getFullYear() + " Muxable";
```

**Review by juaoose — APPROVED** [review]

File: src/component/Test.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Navbar from "./Navbar";
```
Comment (kevmo314) [inline src/component/Test.tsx:1]: Is this component used anywhere/can it be removed?

Comment (KeN095) [reply src/component/Test.tsx:1]: I'll remove it. I was using test to experiment with things but it sounds redundant anyway.

**Review by juaoose — APPROVED** [review]

**Pull request merged**